webknossos-datastore/app/com/scalableminds/webknossos/datastore/helpers/ProtoGeometryImplicits.scala

```diff
@@ -1,10 +1,6 @@
 package com.scalableminds.webknossos.datastore.helpers
 
 import com.scalableminds.util.geometry.{BoundingBox, Vec3Double, Vec3Int}
-<<<<<<< HEAD
```

Comment:
@fm3 as I only had to merge imports I thought I could try that on my own. Could you have a look and make sure this makes sense? (also in `VolumeTracingService.scala`)

Reply:
I added a commit changing one line in each of the two scala files :)

Reply:
thanks for the quick reaction!